.github/workflows/build-ui.yml

```
@@ -34,7 +38,38 @@ jobs:
       working-directory: ui
       run: npm run build
 
+    - name: Commit changes to PR
+      if: github.event_name == 'pull_request'
+      run: |
```

actionlint (1.7.3)

"github.event.head_commit.author.name" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)